src/app/content/components/Attribution.spec.tsx

```diff
@@ -150,5 +150,29 @@ describe('Attribution', () => {
 
       expect(message).toEqual('BUG: Could not find publication date');
     });
+
+    it('displays first two contributing authors when no senior authors were found', async() => {
+      const newState = (cloneDeep({
+        content: {
+          ...initialState,
+          book: formatBookData(book, { ...mockCmsBook,
+            authors: [
+              {value: {name: 'Jhon Doe', senior_author: false}},
+              {value: {name: 'Jonny Doe', senior_author: false}},
+            ],
+          }),
+        },
+      }) as any) as AppState;
+
+      store = createTestStore(newState);
+      const { node } = renderToDom(render());
+      const details = node;
+      details.setAttribute('open', '');
+
+      // wait for dom events to do their thing
+      await new Promise((resolve) => setTimeout(resolve, 1));
```

Comment:
It should work also with `0` instead of `1`
I'm not sure, but maybe there is a way to just await at `await renderToDom(render());` or `renderToDom(await render());`?